zebra-state/src/service/check.rs

```diff
@@ -86,6 +86,41 @@ where
     Ok(())
 }
 
+#[tracing::instrument(
+    name = "contextual_validation_for_chain",
+    fields(?network),
+    skip(prepared, network)
```

Comment:
Tracing style:
1. debugging is easier if we just use the function name
2. Do we need to impl Debug or Display on network, so we can avoid skipping it?
```suggestion
    skip(prepared)
```

Reply:
To be frank I just copied the other function in the module, I'll eventually read a bit more about `tracing::instrument` to understand this better.

Reply:
Ha yeah we should clean them both up, I didn't know much about tracing when I wrote it.

Reply:
Done!